plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/dataimport/CommonElementImporter.java

```diff
@@ -439,7 +439,7 @@ protected void parseGenericAttributeNode(final ConfigurableObject confObject)
 						name -> getTypeLibrary().getAttributeTypeEntry(name), name -> null);
 				final AttributeTypeEntry attributeTypeEntry = addDependency(entry);
 				if (attributeTypeEntry != null && attributeTypeEntry.getType() != null) {
-					attribute.setAttributeDeclaration(attributeTypeEntry.getType());
+					attribute.setAttributeDeclaration(addDependency(attributeTypeEntry.getType()));
```

Comment:
There is already an `addDependency(...)` in line 440.

Reply:
Damn missed that. Fixed.